contrib/core-contract-tests/tests/pox-4/pox_StackAggregationCommitAuthCommand.ts

```diff
@@ -112,6 +112,7 @@ export class StackAggregationCommitAuthCommand implements PoxCommand {
 
     const operatorWallet = model.wallets.get(this.operator.stxAddress)!;
     operatorWallet.amountToCommit -= committedAmount;
+    operatorWallet.committedRewCycleIndexes.push(model.nextRewardSetIndex);
```

Comment:
If the wallet is using more than one pox address the index storage should be a map of pox addresses to a list of indexes.